docs/merged_objects.md

```diff
@@ -393,3 +393,85 @@ merged_adata_object.var # gene metrics
+```
+
+Column names are cell barcode sequences prefixed with the originating library id, e.g. `SCPCL000000-{barcode}`, and row names are the ADT IDs.
+
+```python
+merged_citeseq_adata_object.obs_names # matrix column names
+merged_citeseq_adata_object.var_names # matrix row names
+```
+
+All of the per-cell data columns included in the `colData` of the `"adt"` alternative experiment in `SingleCellExperiment` merged objects are present in the `.obs` slot of the CITE-seq `AnnData` object.
+To see a full description of the included columns, see the section [on additional `SingleCellExperiment` components for CITE-seq libraries](#additional-singlecellexperiment-components-for-cite-seq-libraries-with-adt-tags).
+
+
+The following QC statistics from [`DropletUtils::cleanTagCounts()`](https://rdrr.io/github/MarioniLab/DropletUtils/man/cleanTagCounts.html) can be found in the `obs` slot of each ADT-specific `AnnData` object.
+
+| Column name                | Contents                                          |
+| -------------------------- | ------------------------------------------------- |
+| `zero.ambient`   | Indicates whether the cell has zero ambient contamination   |
+| `sum.controls` |  The sum of counts for all control features. Only present if negative/isotype control ADTs are present |
+| `high.controls`  | Indicates whether the cell has unusually high total control counts. Only present if negative/isotype control ADTs are present |
+| `ambient.scale` |  The relative amount of ambient contamination. Only present if negative control ADTs are _not_ present |
+| `high.ambient`  | Indicates whether the cell has unusually high contamination. Only present if negative/isotype control ADTs are _not_ present |
+| `discard`       | Indicates whether the cell should be discarded based on QC statistics. The `TRUE` and `FALSE` values in this column correspond, respectively, to values `"Discard"` and `"Keep"` in the `adata_obj.obs["adt_scpca_filter"]` column                                                         |
+
+Metrics for each of the ADTs assayed can be found as a `pandas.DataFrame` in the `.var` slot within the CITE-seq `AnnData` object:
+
+| Column name | Contents                                                       |
+| ----------- | -------------------------------------------------------------- |
+| `adt_id`  | Name or ID of the ADT                                              |
+| `mean-SCPCL000000`      | Mean ADT count across all cells/droplets. Only present for libraries with CITE-seq data                       |
+| `detected-SCPCL000000`  | Percent of cells in which the ADT was detected (ADT count > 0 ). Only present for libraries with CITE-seq data |
+| `target_type` | Whether each ADT is a target (`target`), negative/isotype control (`neg_control`), or positive control (`pos_control`). If this information was not provided, all ADTs will have been considered targets and will be labeled as `target` |
```

Comment:
Same comment, do we need to have the same table twice?

Reply:
Hmm I see in the SCE contents we present it like this. I guess it's fine to just repeat the tables, but we don't do that for any other section. Every other place we just link to the SCE contents.

Reply:
> Hmm I see in the SCE contents we present it like this. I guess it's fine to just repeat the tables, but we don't do that for any other section. Every other place we just link to the SCE contents.

Ha, just getting to this comment after replying about it https://github.com/AlexsLemonade/scpca-docs/pull/262#discussion_r1476474427
I think we should remove the tables and just link!

Reply:
Yes, I'm on board with this plan!